opengever/maintenance/browser/enable_sqlalchemy_logging_filter.py

```diff
@@ -0,0 +1,58 @@
+from five import grok
+from opengever.ogds.base.utils import create_session
+from Products.CMFPlone.interfaces import IPloneSiteRoot
+from threading import RLock
+import logging
+
+
+# Module globals to track whether the next bound SQL parameters should be
+# logged or not, and the according lock to ensure thread safety
+_log_next_param = False
+_log_next_param_lock = RLock()
```

Comment:
I don't understand why we need to share this information globally, i.e. between threads? If not i think it would be better to use threading.local? e.g.:

```
logging_state = threading.local()

...
class TaskUpdateFilter(logging.Filter):
    def __init__(self):
        logging_state.log_next_param = False

    def filter(record):
        if logging_state.log_next_param and TaskUpdateFilter.is_param_entry(record):
            logging_state.log_next_param = False
            return True
    ...

```

*untested code, but something like that should work :wink:*